entry_check.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
should prolly delete this file from the commit but im assuming its cos PHP storm asks if it shoudl be added lol